Sources/CardPayments/CardClient.swift

```diff
@@ -50,9 +50,7 @@ public class CardClient: NSObject {
                 } else {
                     analyticsService?.sendEvent("card-payments:3ds:confirm-payment-source:succeeded")
                     
-                    let cardResult = CardResult(
-                        orderID: result.id
-                    )
+                    let cardResult = CardResult(orderID: result.id, deepLinkURL: nil)
```

Comment:
We want to be able to return order status here, right? I think the issue was that after 3ds challenge, we didn't have updated order status so we just deleted that field in CardResult for now. I think we definitely want to add that order status field back, and assume some status, or return "UNKNOWN" or something after 3DS challenge without error.

Reply:
Looks like both `paymentSource` and `status` was removed on the [Android PR](https://github.com/paypal/Android-SDK/pull/146). We should do the same on iOS so the platforms are aligned. There was a conversation here about this decision: https://github.com/paypal/Android-SDK/pull/146#discussion_r1210503077